app/src/main/java/com/iitp/njack/iitp_connect/common/di/modules/DaoModule.java

```diff
@@ -30,12 +31,14 @@ DatabaseReference providesFirebaseDatabaseReference() {
         return FirebaseDatabase.getInstance().getReference();
     }
 
-    @Singleton @Provides
+    @Singleton
+    @Provides
     public UserDao providesUserDao(IITPConnectDatabase iitpConnectDatabase){
         return iitpConnectDatabase.userDao();
     }
 
-    @Singleton @Provides
+    @Singleton
+    @Provides
```

Comment:
The changes in this file are unrelated to the PR. Please don't make unrelated changes. It makes it difficult to find the source of bugs later on.

Reply:
You can create a new issue for and do it. Please revert these unnecessary changes in this PR.

Reply:
You can create a new issue for and do it. Please revert these unnecessary changes in this PR.

Reply:
You can create a new issue for and do it. Please revert these unnecessary changes in this PR.